### PR TITLE
Error style section

### PR DESCRIPTION
**Pull request by JoaquinTrinanes**
Related to https://github.com/nushell/nushell/pull/10122, this PR creates an accessibility subsection explaining recommended settings for screen reader users.

### CODE REVIEW

File: book/coloring_and_theming.md [hunk 1]
```diff
@@ -517,3 +517,20 @@ $env.config = {
   ...
 }
 ```
+
+## Accessibility
+
+It's often desired to have the minimum amount of decorations when using a screen reader. In those cases, it's possible to disable borders and other decorations for both table and errors with the following options:
+
+```shell
+# in $nu.config-file
```
Comment (amtoine) [inline book/coloring_and_theming.md:526]:
```suggestion
# in $nu.config-path
```
:wink:

**Review by amtoine — COMMENTED** [review]
thanks a lot :pray: 

let's wait for nushell/nushell#10122 and then i think we can land this :ok_hand:

File: book/coloring_and_theming.md (hunk 1 above)
Comment (amtoine) [reply book/coloring_and_theming.md]: ooh there were more `$nu.config-file` in that page :scream: 
thanks for fixing the other ones :pray:

Comment (JoaquinTrinanes) [reply book/coloring_and_theming.md]: No problem! They were the reason I wrote that in the first place 😌

**Comment by JoaquinTrinanes** [comment]
I'm wondering if we should add indications to disable the banner in the accessibility section, to avoid all the noise caused by
```
     __  ,
 .--()°'.' Welcome to Nushell,
'|, . ,'   based on the nu language,
 !_-(_\    where all data is structured!
```

**Comment by fdncred** [comment]
Thanks! Let's leave the banner for now.

**Pull request merged**